src/web/layouts/DecideLayout.tsx

```diff
@@ -13,7 +12,7 @@ type Props = {
 };
 
 export const DecideLayout = ({ designType, CAPI, NAV }: Props) => {
-    if (hasShowcase(CAPI.mainMediaElements)) {
+    if (CAPI.pageType.hasShowcaseMainElement) {
```

Comment:
Would you have a similar check here for immersive `displayHint`s once you've removed "Immersive" from `DesignType`?

Reply:
Yes, exactly. Once we've got an `ImmersiveLayout` built we can start using it like that. The storybook story that this ticket adds is essentially visual TDD.